### DISCUSSION TITLE
I'm search a plataform no-code

### DISCUSSION THREAD

**Discussion by carusoalexandre** [Budibase/budibase#1873] [Help] [answered]
Guys, I developed an app in GlideApps to help poor people to pay their basic bills, light, gas, or water.
Now I'm searching a site that I could build a website version (let´s say) of this app.

It means I could build just dragging and drop elements, set proprietaries, connect to Zapier or Automato.io, API first...
Everything that I wouldn´t need a dev. For instance add login screen, connect and list record in the database on Azure or AWS.

Is the budibase that platform?
I accept suggestion.
Thx in advance

**Comment by shogunpurple** [comment, marked as answer]
@carusoalexandre Budibase is a platform generally used for building internal tools within an organisation. 

To address your points:
- You could use budibase for your use case, to fetch data from your database and display inside your apps. Budibase has a strong SQL integration that will allow you to do this with ease
- Drag and drop is not something that budibase does at the moment, but you'll find more information around the rationale of that decision here: https://github.com/Budibase/budibase/discussions/1819
- You can use the built in budibase login screen for authenticating with your app. This is opinionated by default, and does not have a huge amount of customisation options
- We have automation integrations with Zapier and Integromat, but not Automato.io.

Coming from glide, you will find budibase to be a more developer focused platform. We are a low code development platform that augments the development experience, rather than a fully no-code solution to replace it. 

Hope this helps!

**Reply by carusoalexandre** [reply]
Sorry just one more issue, is it free? or freemium? Thx again.

**Reply by shogunpurple** [reply]
@carusoalexandre all the features you mentioned are in the free and open source version of budibase. There will be paid features and options which are more aimed at larger organisations in future.
